### PR TITLE
Create blog

### PR DESCRIPTION
**Pull request by JessSaavedra**
## Summary

- Added files for simple blog

## Trello Card

https://trello.com/c/QcL7S2Ts

### CODE REVIEW

File: app/assets/javascripts/comments.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (epintos) [inline app/assets/javascripts/comments.coffee]: Be carefull with all the autogenerated files. The repository shouldn't have files that are not used at all

File: app/helpers/comments_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module CommentsHelper
```
Comment (epintos) [inline app/helpers/comments_helper.rb]: Same for not used helpers

**Comment by epintos** [comment]
Lets remove the unused files in another branch to avoid git conflicts

:+1:

**Comment by JessSaavedra** [comment]
Fixed in "Delete unused files" PR

**Pull request merged**